spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -242,6 +242,8 @@ public void onMessage(ConsumerRecord<Integer, String> message) {
 	public void testManualCommit() throws Exception {
 		testManualCommitGuts(AckMode.MANUAL, topic4);
 		testManualCommitGuts(AckMode.MANUAL_IMMEDIATE, topic5);
+		testManualCommitGuts(AckMode.MANUAL, topic4);
+		testManualCommitGuts(AckMode.MANUAL_IMMEDIATE, topic5);
```

Comment:
I wanted to be sure the commits worked ok so I ran the tests again and the second tests start at the committed offset.